corehq/apps/locations/permissions.py

```diff
@@ -1,3 +1,77 @@
+"""
+Location Permissions
+====================
+
+Normal Access
+-------------
+
+Location Types - Users who can edit apps on the domain can edit location types::
+Locations - Users who can edit mobile workers on the domain can edit locations
+
+
+Restricted Access and Whitelist
+--------------------------------
+
+Many large projects have mid-level users who should have access to a subset of
+the project based on the organization's hierarchy.
+
+This is handled by a special permission called "Full Organization Access" which
+is enabled by default on all user roles. To restrict data access based on a
+user's location, projects may create a user role with this permission disabled.
+
+This is checked like so::
+
+    user.has_permission(domain, 'access_all_locations')
```

Comment:
That's a good point.  Farid and I just spoke about this and he pointed out that we already use the term "location" everywhere else in the code, so I think it makes sense to stick with that, so at least the code is consistent.